ccache.c

```diff
@@ -1053,6 +1064,21 @@ to_cache(struct args *args)
 		tmp_cov = NULL;
 	}
 
+	char *tmp_su;
+	if (generating_stackusage) {
+                char *tmp_aux;
+                // GCC has some funny rule about max extension length.
```

Comment:
I'm wondering if this applies to stackusage the way it did for coverage... (gcno)

Really should dig down the GCC source code that generates these someday...